platform/node/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 
 ## main
 
+* ...Add new stuff here...
```

Comment:
Personally I would not duplicate changelog until we start making `main` and `opengl-2` releases in parallel.

Reply:
I got rid of the duplication in main. 

the two things that we under 'main' before were part of the opengl-2 branch already, so this is actually the first release they will be included in.

Reply:
I was thinking though that after the release, that maybe we changelog and package.json changes might be good to cherry pick back to main so everything is in sync version and changelog wise.